shared/chat/conversation/messages/ids-context.tsx

```diff
@@ -0,0 +1,4 @@
+import * as React from 'react'
+import type * as Types from '../../../constants/types/chat2'
+export const ConvoIDContext = React.createContext<Types.ConversationIDKey>('')
```

Comment:
we now use a context to avoid plumbing the ids everywhere

Reply:
this is primarily for perf as the middling components don't have to rerender all the time (aka the ordinal might change but the left area will be blank, yet all the middle components will rerender to plumb the ordinal down) and its also less plumbing